.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url: https://discord.gg/tokio
```

Comment:
The README includes a link directly to the channel: https://discord.gg/RFsPjyt

Reply:
Excellent, I'll use that then!

Reply:
Done, seems to work fine.